tools/fast/fasting/basic.go

```diff
@@ -0,0 +1,132 @@
+package fasting
```

Comment:
:thinking: not the biggest fan of this name, but I don't have a better idea, `fastesting`?